radar/infrastructure_config/kubernetes.yaml

```diff
@@ -0,0 +1,19 @@
+version: 1
+name: Kubernetes
+blip:
+  - version: "2.0"
```

Comment:
This should be changed to the `MAJOR.MINOR` version in which the blip is moved. In this case, the current radar version is `2.2.1` so the value should be `2.3.`.